motionplan/armplanning/node.go

```diff
@@ -92,30 +98,34 @@ func extractPath(startMap, goalMap rrtMap, pair *nodePair, matched bool) []*node
 	return path
 }
 
-// This function is the entrypoint to IK for all cases. Everything prior to here is poses or configurations as the user passed in, which
-// here are converted to a list of nodes which are to be used as the from or to states for a motionPlanner.
-func generateNodeListForPlanState(
+// This function is the entrypoint to IK for all cases. Everything prior to here is poses or
+// configurations as the user passed in, which here are converted to a list of nodes which are to be
+// used as the from or to states for a motionPlanner.
+func generateNodeListForGoalState(
 	ctx context.Context,
-	mp *cBiRRTMotionPlanner,
-	state *PlanState,
+	motionPlanner *cBiRRTMotionPlanner,
```

Comment:
I might have gone a step too far with this local variable renaming. There was a time where I was reading `mp` as "map" and thinking `rrtMap`.